cms/djangoapps/contentstore/views/videos.py

```diff
@@ -328,6 +331,13 @@ def videos_post(course, request):
     for req_file in req_files:
         file_name = req_file["file_name"]
 
+        # Check if file_name contains non-ascii characters.
+        try:
+            file_name.encode('ascii')
+        except UnicodeEncodeError as exception:  # pylint: disable=unused-argument
+            error_msg = "%s contains non ascii characters in file name." % file_name
```

Comment:
I think should se the error notification. It has two parts, 1) title; which is generic to video uploads. 2) error message :

<img width="666" alt="screen shot 2016-12-16 at 10 12 01 pm" src="https://cloud.githubusercontent.com/assets/6991154/21271417/c131249c-c3dc-11e6-97e3-ec36d803b00d.png">

So I am not sure if that would be correct to display `could not be uploaded` two times ? Just wanted to make this sure, there is title text as well :)

Or we can add file name in the title field too. What is you thoughts on that?